styles/global.scss

```diff
@@ -371,3 +371,75 @@ hr {
     }
   }
 }
+
+.dialog {
```

Comment:
can this css be contained inside a component?

Reply:
this component is changing, I'll change this pr to draft for now